solutions/src/test/java/ru/sberbank/school/task10/ScalableThreadPoolTest.java

```diff
@@ -0,0 +1,115 @@
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class ScalableThreadPoolTest {
+    private ThreadPool threadPool;
+
+    @Before
+    public void init() {
+        threadPool = new ScalableThreadPool(3, 10);
+        threadPool.start();
+    }
+
+    @Test(timeout = 400)
+    public void normalWork() throws InterruptedException {
+        final AtomicInteger counter = new AtomicInteger(0);
+        for (int i = 0; i < 25; i++) {
+            threadPool.execute(() -> {
+                try {
+                    TimeUnit.MILLISECONDS.sleep(40);
+                } catch (InterruptedException ignore) {
+//                    Thread.currentThread().interrupt();
+                }
+                synchronized (counter) {
+                    System.out.println("Task: " + counter.incrementAndGet()
+                            + " - Thread: " + Thread.currentThread().getName());
+                }
+            });
+        }
+        TimeUnit.MILLISECONDS.sleep(100);
+        for (int i = 0; i < 25; i++) {
+            TimeUnit.MILLISECONDS.sleep(6);
+            threadPool.execute(() -> {
+                try {
+                    TimeUnit.MILLISECONDS.sleep(5);
+                } catch (InterruptedException ignore) {
+//                        Thread.currentThread().interrupt();
+                }
+                synchronized (counter) {
+                    System.out.println("Task: " + counter.incrementAndGet()
+                            + " - Thread: " + Thread.currentThread().getName());
+                }
+            });
+        }
+        TimeUnit.MILLISECONDS.sleep(50);
+        Assertions.assertEquals(50, counter.get());
+        threadPool.stopNow();
+    }
+
+    @Test(timeout = 400)
+    public void callableService() throws InterruptedException, ExecutionException {
+        final List<Future> futureList = new ArrayList<>();
+        final AtomicInteger counter = new AtomicInteger(0);
+        for (int i = 0; i < 25; i++) {
+            Future<String> future = threadPool.execute(() -> {
+                try {
+                    TimeUnit.MILLISECONDS.sleep(100);
+                } catch (InterruptedException ignore) {
+//                        Thread.currentThread().interrupt();
+                }
+                String res;
+
+                synchronized (counter) {
+                    res = "Task: " + counter.incrementAndGet() + " - Thread: " + Thread.currentThread().getName();
+                }
+                return res + " " + LocalDateTime.now().toString();
+
+            });
+            futureList.add(future);
+        }
+        for (Future future : futureList) {
+            System.out.println(future.get());
+        }
+        Assertions.assertEquals(25, counter.get());
+        threadPool.stopNow();
+    }
+
+
+    @Test(expected = IllegalStateException.class)
+    public void executeAfterStop() {
+        threadPool.stopNow();
+        threadPool.execute(() -> {});
```

Comment:
WhitespaceAround: &#39;}&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-971656239>*